app/views/stafftools/users/_user.html.erb

```diff
@@ -5,5 +5,8 @@
     <strong class="search-result-title"><%= user.login %></strong>
   <% end %>
 
-  <span class="text-muted css-truncate css-truncate-target">Joined <%= local_time_ago(user.created_at) %></span><br>
+  <span class="text-muted css-truncate css-truncate-target">
+    Joined <%= local_time_ago(user.created_at) %> /
+    Last active: <%= user.last_active_at.present? ? local_time_ago(user.last_active_at) : 'Unknown' %>
```

Comment:
Why don't we set the `last_active_at` to `created_at` when the user is first created? That way we never have the case where the `last_active_at` is unknown.